back/sssscs/src/main/java/com/ib/certificate/CertificateController.java

```diff
@@ -28,25 +28,21 @@
 public class CertificateController {
 	@Autowired
 	private ICertificateService certificateService;
-	
 	@Autowired
 	private ICertificateRequestService certificateRequestService;
-	
 	@Autowired
 	private IUserService userService;
-	
 	@Autowired
 	private ModelMapper modelMapper;
 	
 	@PostMapping("/request")
-	public ResponseEntity<String> makeRequest(/*@DTO(CertificateRequestCreateDto.class)*/ @RequestBody CertificateRequestCreateDto certificate) {
-		// TODO: Automatic mapping.
-		
+	public ResponseEntity<String> makeRequest(@RequestBody CertificateRequestCreateDto certificate) {
 		CertificateRequest req = new CertificateRequest();
```

Comment:
Seems to be missing a validation annotation? I think it might be required if you are not using @DTO? Maybe? Might be wrong

Reply:
What do you mean?